### ISSUE TITLE
v4: Xcode hangs when bundling

### ISSUE THREAD

**Issue by johnf** [nativewind/nativewind#589]
**Describe the bug**
When building a dev version of the app Xcode doesn't exit.

**To Reproduce**
Create a dev build with XCode

**Expected behavior**
The build should complete

**Expo Snack**
N/A

**Screenshots**
N/A

**Additional context**
I've narrowed this down to https://github.com/marklawlor/nativewind/blob/main/packages/nativewind/src/metro/tailwind-cli.ts#L58-L77

Xcode runs the equivalent of (I've changed the [aths for legibility
```sh
node node_modules/react-native/cli.js bundle --entry-file index.js --platform ios --dev true  --bundle-output main.jsbundle --assets-dest Gladly-dev.app --sourcemap-output main.jsbundle.map
```

In the code above a web socket is started due to the --dev true flag.

This means that the bundle command hangs and never exits.

I'm not sure what a good solution would be here. I can't find any obvious way to work out if the metro code was called from cli.js bundle or start

**Comment by johnf** [comment]
I worked around this for CI at least by doing this
```diff
iff --git a/node_modules/nativewind/dist/metro/tailwind-cli.js b/node_modules/nativewind/dist/metro/tailwind-cli.js
index 6631649..edc018b 100644
--- a/node_modules/nativewind/dist/metro/tailwind-cli.js
+++ b/node_modules/nativewind/dist/metro/tailwind-cli.js
@@ -35,7 +35,7 @@ async function tailwindCli(input, options) {
     let version = 0;
     let startedWSServer = false;
     const connections = new Map();
-    if (options.dev) {
+    if (!process.env.CI && options.dev) {
         spawnCommands.push("--watch");
         if (options.platform !== "web") {
             startedWSServer = true;
```

We could probably find a similar env variable that Xcode sets but it feels a bit hacky

**Comment by johnf** [comment]
Fixed by #63c6a6a3a57d8bc2caf5945a2f775834f16b4287
